app/controllers/audios_controller.rb

```diff
@@ -20,6 +20,7 @@ def new
     @formdef = PSSBrowserUploads.av_form_definition('audio')
     @accepted_types = %w(.mp3 .mp2 .mp4a .wav .flac .aif .aiff .wma .mpga .oga
                          .ogg .au .adp .aac .weba).join(',')
+    @source = Source.find_by_id(params[:source_id])
```

Comment:
OK, that's cool.

It's out of scope here, but one thing that confuses me is:
```
[2] pry(main)> Source.find_by_id(99)
  Source Load (1.4ms)  SELECT  "sources".* FROM "sources"  WHERE "sources"."id" = 99 LIMIT 1
=> nil
[3] pry(main)> Source.find(99)
  Source Load (1.5ms)  SELECT  "sources".* FROM "sources"  WHERE "sources"."id" = ? LIMIT 1  [["id", 99]]
ActiveRecord::RecordNotFound: Couldn't find Source with 'id'=99
from /home/vagrant/.rbenv/versions/2.1.3/lib/ruby/gems/2.1.0/gems/activerecord-4.1.13/lib/active_record/relation/finder_methods.rb:320:in `raise_record_not_found_exception!'
[4] pry(main)> 
```
That inconsistency isn't what I'd expected, and I thought that `find_by_id` would also have thrown that exception.  It's another topic, though, not a problem here and now.